packages/graphicsmagick.rb

```diff
@@ -32,6 +19,7 @@ def self.build
     system './configure',
            "--prefix=#{CREW_PREFIX}",
            "--libdir=#{CREW_LIB_PREFIX}",
+           "--mandir=#{CREW_PREFIX}/share/man",
```

Comment:
This line could be: `--mandir=#{CREW_MAN_PREFIX}` but you could probably replace all 3 lines above with `#{CREW_OPTIONS}`.